metricflow/query/query_parser.py

```diff
@@ -397,7 +399,17 @@ def _parse_and_validate_query(
         where_constraint_str: Optional[str] = None,
         order_by_names: Optional[Sequence[str]] = None,
         order_by: Optional[Sequence[OrderByQueryParameter]] = None,
+        min_max_only: bool = False,
```

Comment:
@courtneyholcomb please coordinate with @plypaul , this file changed dramatically in his pending stack of changes.